spec/matchers/log.rb

```diff
@@ -0,0 +1,11 @@
+RSpec::Matchers.define :log do |job_output|
```

Comment:
While my example in the README does test for output, I'm not sure how useful this matcher would be in practice.

I would guess (though I'm not sure) that the number of jobs that have output, or more specifically a string response, may be minimal, and the value of testing that may be small.

The intention of the `block_for` method is to provide full access to whatever the job is doing, to run future assertions against it. So, if you had a job that gave all users in the system 100 points, you would test that 1) the job ran 2) that users had 100 more points than they did before. A contrived example for sure, but something like: 

```ruby
before do
  User.first.update_attributes(points: 0)
  Clockwork::Test.run(file: clock_file, max_ticks: 1)
  Clockwork::Test.block_for("give everyone points").call
end

after { Clockwork::Test.clear! }

it "gives a user 100 points" do
  expect(User.first.points).to eq 100
end
```